v1/src/test/java/com/google/cloud/teleport/spanner/ExportPipelineIT.java

```diff
@@ -44,14 +46,35 @@
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
 
 /** Integration test for {@link ExportPipeline Spanner to GCS Avro} template. */
 @Category(TemplateIntegrationTest.class)
 @TemplateIntegrationTest(ExportPipeline.class)
-@RunWith(JUnit4.class)
+@RunWith(Parameterized.class)
 public class ExportPipelineIT extends TemplateTestBase {
 
+  private String spannerHost;
+  private String spannerHostName;
+
+  public ExportPipelineIT(String spannerHost, String spannerHostName) {
+    this.spannerHost = spannerHost;
+    this.spannerHostName = spannerHostName;
+  }
+
+  @Parameters(name = "{1}")
+  public static Collection primeNumbers() {
```

Comment:
Why is this method called `primeNumbers` o.O?

Reply:
Corrected. copy paste mistake :)